django/santropolFeast/member/templates/client/view/status.html

```diff
@@ -0,0 +1,69 @@
+{% extends "client/base.html" %}
+<!-- Load internationalisation utils-->
+{% load i18n %}
+{% load admin_urls %}
+
+{% block title %}
+
+  {{client}} - Information
+
+{% endblock %}
+
+{% block subcontent %}
+
+<div class="ui segment">
+    <h2 class="ui header">
+        <div class="content">
+            {% trans 'Scheduled status modifications' %}
+            <div class="sub header">
+                {% trans 'Here are listed all scheduled status modifications.' %}
+            </div>
+        </div>
+    </h2>
+    <div class="ui divider"></div>
+
+    <form action="" method="GET" class="ui form">
+        <div class="inline fields">
+            <div class="field">
+                <label>{% trans 'Operation status' %}</label>
+                {{ filter.form.operation_status }}
```

Comment:
@philippemouchel copy/paste error, sorry. {% %} is for call to function tags like 'trans', {{ }} is for print values..